src/pages/workspace/companyCards/assignCard/AssignCardFeedPage.tsx

```diff
@@ -27,7 +27,14 @@ function AssignCardFeedPage({route, policy}: AssignCardFeedPageProps) {
     const backTo = route.params?.backTo;
     const policyID = policy?.id;
     const [isActingAsDelegate] = useOnyx(ONYXKEYS.ACCOUNT, {selector: (account) => !!account?.delegatedAccess?.delegate});
-
+    const firstAssigneeEmail = useRef(assignCard?.data?.email);
+    // eslint-disable-next-line react-compiler/react-compiler
+    if (!firstAssigneeEmail.current) {
+        // eslint-disable-next-line react-compiler/react-compiler
+        firstAssigneeEmail.current = assignCard?.data?.email;
+    }
+    // eslint-disable-next-line react-compiler/react-compiler
+    const shouldUseBackToParam = !firstAssigneeEmail.current || firstAssigneeEmail.current === assignCard?.data?.email;
```

Comment:
Instead of disabling the lint rule for each line, let's disable it once and then reenable after `shouldUseBackToParam` is assigned. (Use `eslint-disable` / `eslint-enable` instead of `eslint-disable-next-line`)